### DISCUSSION TITLE
Difference between Ubuntu and Ubuntu GTK3 builds?

### DISCUSSION THREAD

**Discussion by blex-max** [supermerill/SuperSlicer#2170] [Q&A] [answered]
Quick, probably simple, question - what is the difference between the Ubuntu and the Ubuntu GTK3 nightly builds? I've always used the GTK3 build since I'm familiar with GTK and it's the version available in releases but the interface for superslicer is a little buggy so I am curious as to what the other option is.

**Comment by supermerill** [comment, marked as answer]
it's GTK2
I had some bug report that are fixed by using the GTK2 one, so it's still the "default one".
